c2corg_images/__init__.py

```diff
@@ -3,15 +3,19 @@
 from pyramid.events import NewRequest
 import os
 
+
 # See http://docs.wand-py.org/en/0.4.1/guide/resizecrop.html
 # Max 800, keep aspect ratio
 # Max 120x120 don't keep ratio
 THUMBNAIL_CONFIGS = [
-    {'suffix': 'BI', 'convert': ['-resize', '800x800>']},
-    {'suffix': 'MI', 'convert': ['-resize', '250x250>']},
+    {'suffix': 'BI', 'convert': ['-resize', '800x800>',
+                                 '-quality', '80']},
```

Comment:
Is 80 really enough to get a satisfaying image quality?

In v5 the resized images quality is poor whereas using ``convert`` with quality ranging from 75 to 85 it seems:
https://github.com/c2corg/camptocamp.org/blob/72e777075f2e6260b1d2ef563ab0f90c383038ba/apps/frontend/lib/Images.class.php#L159
https://github.com/c2corg/camptocamp.org/blob/72e777075f2e6260b1d2ef563ab0f90c383038ba/plugins/sfThumbnailPlugin/lib/sfThumbnail.class.php#L43
https://github.com/c2corg/camptocamp.org/blob/72e777075f2e6260b1d2ef563ab0f90c383038ba/apps/frontend/lib/Images.class.php#L47-L48

What about using quality = 90 for instance?